scripts/internal/workflow-helpers/jira.go

```diff
@@ -97,53 +103,53 @@ func FetchAvailableVersions(jiraClient *jira.Client) ([]semver.Version, error) {
 			logging.Debug("Not a semver version %s; skipping", version.Name)
 			continue
 		}
-		result = append(result, semversion)
+		result = append(result, Version{semversion, version.ID})
 	}
 
 	sort.Slice(result, func(i, j int) bool {
-		return result[i].LT(result[j])
+		return result[i].LT(result[j].Version)
 	})
 
 	return result, nil
 }
 
 var VersionMaster = semver.MustParse("0.0.0")
 
-func ParseTargetFixVersion(issue *jira.Issue, availableVersions []semver.Version) (semver.Version, *jira.FixVersion, error) {
+func ParseTargetFixVersion(issue *jira.Issue, availableVersions []Version) (target Version, original *jira.FixVersion, err error) {
```

Comment:
The return value names do not appear to be used. Perhaps you can remove them?

Reply:
I meant the named return values `target`, `original`, and `err`, not the actual return values. You are not setting `target`, `original`, or `err`, but returning actual values everywhere in this function. Am I misunderstanding named return values in Go?

Reply:
Oh I see. I named it so that the API is self-explanatory. I found that without the naming I was interpreting the second argument as being the same as the first but wrapped in jira.FixVersion, whereas in actuality it's the jira fixVersion that it used to calculate the returned version from.

So the point of naming in this context is documentation. I'm not looking to use those variables functionally.